api/python/quilt3/workflows/__init__.py

```diff
@@ -294,11 +294,17 @@ def validate(*, registry: PackageRegistry, workflow, name, pkg, message):
     if not (workflow in (None, ...) or isinstance(workflow, str)):
         raise TypeError
 
-    workflow_config = registry.get_workflow_config()
-    if workflow_config is None:
-        if workflow is ...:
+    try:
+        workflow_config = registry.get_workflow_config()
+        if workflow_config is None:
+            if workflow is ...:
+                return
+            raise util.QuiltException(f'{workflow!r} workflow is specified, but no workflows config exist.')
+    except Exception as e:
```

Comment:
the exception is too broad and logic is not clear
`registry.get_workflow_config()` already returns `None` if workflow config doesn't exist
IMO we should fail in case (and other cases?) default workflow is to be used but workflow config can't be read because of lacking permissions or network error

Reply:
Reverting, not needed.